Dockerfile

```diff
@@ -30,13 +30,15 @@ WORKDIR /iroha
 COPY . .
 RUN cargo chef prepare --recipe-path recipe.json
 
-ARG PROFILE=
+ARG PROFILE=""
 FROM cargo-chef as cacher
+ARG PROFILE
```

Comment:
I guess it shouldn't repeat twice. (Also `PROFILE=` and `PROFILE=""` has same semantics)

Reply:
Actually it is enough to redeclare arg into build stage after "cacher" build stage. I redeclared it twice to make behaviour obvious.

Reply:
Seems strange to see almost same line within range of 3 lines

Reply:
Ok, I will make it less verbose